src/icon-manifest/iconGenerator.ts

```diff
@@ -80,6 +80,18 @@ export class IconGenerator implements models.IIconGenerator {
     }
   }
 
+  public getCustomIconFolderPath(folderPath: string): string {
+    if (!!!folderPath) { return this.settings.vscodeAppData; }
```

Comment:
One `!` should be enough.

Reply:
I'm converting them to `boolean`.

Reply:
I know, that's what I say you only need one

Reply:
Actually one is a null check. Three is a boolean negative.

Reply:
There's no such thing as a null check in `JavaScript`. `!` is a logical operator and also implies a casting to `bool` due to the concepts of falsy and truthy. `!!!` is not idiomatic JavaScript.

Reply:
https://stackoverflow.com/posts/25318045/revisions

Reply:
It's just my preferred way of coding, expressing the fact that I'm doing `!!(!a)`.

Reply:
It's ok but it's not really necessary and thus it's not used in JavaScript. I would prefer to get rid of this if possible as I think we should stick to common practices. 😉